pyobis/__init__.py

```diff
@@ -34,7 +34,11 @@
 occurrence.search(geometry='POLYGON((30.1 10.1, 10 20, 20 40, 40 40, 30.1 10.1))')
 """
 
-__version__ = "1.2.5"
```

Comment:
With the lines below, when we create a tag on GitHub, a `_version.py` file will be written and the installation will read the version from it. That way we need to change the version in a single place, the tag, and everything else is automatic.

Reply:
beautiful. I am learning so many great tricks through this project. Thanks!

Reply:
Sadly all I can teach is packaging tricks. At least I can save you two from the packaging hell so you can focus on fun coding :-)

Reply:
I am actually having fun coding on this project. And thank you for keeping me out of those trappy things so that I would keep doing that fun :)